CMakeLists.txt

```diff
@@ -393,7 +393,7 @@ if(USE_THREADS AND NOT BUILD_FOR_HEXAGON)
   target_link_libraries(tvm_runtime Threads::Threads)
 endif()
 
-target_link_libraries(tvm ${TVM_LINKER_LIBS} ${TVM_RUNTIME_LINKER_LIBS})
+target_link_libraries(tvm -static-libgcc ${TVM_LINKER_LIBS} ${TVM_RUNTIME_LINKER_LIBS})
```

Comment:
Can append() the linker flag to TVM_LINKER_LIBS variable using condition on cmake's CMAKE_SYSTEM_PROCESSOR is arm64 ?